<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsbPortPicker</class>
 <widget class="QWidget" name="UsbPortPicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="usbGroup">
     <property name="title">
      <string>USB connection</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="usbPortComboBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="usbDisconnectPushButton">
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="usbConnectPushButton">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
